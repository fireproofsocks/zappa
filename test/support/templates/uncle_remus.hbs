Wo, are we movin' too slow?
Have you seen us,
{{{father_figure}}}...