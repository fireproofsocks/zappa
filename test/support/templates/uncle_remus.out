Wo, are we movin' too slow?
Have you seen us,
<bold>Charismatic Black Man</bold>...